src/resolvers/block.ts

```diff
@@ -89,6 +89,33 @@ const blocksRange: IFieldResolver<any, any> = async (obj, { numberRange, hashRan
 const transactions = (obj, args) =>
   obj.transactions.map(tx => ({ ...tx, from: { address: tx.from }, to: { address: tx.to }, inputData: tx.input }));
 
+interface ITransactionsInvolvingArgs {
+  participants?: [string];
+}
+  
+const transactionsInvolving: IFieldResolver<any, any> = async (obj, { participants }: ITransactionsInvolvingArgs) => {
```

Comment:
I see that this makes a JSON-RPC request for each transaction in the block. Given that `obj` is already a block that came from `web3.eth.getBlock(..., true)`, and hence it contains all transaction data (`obj.transactions`), would it not be more efficient to traverse the transactions we already have, and select only the ones that match the criteria? You should be able to achieve with `obj.transactions.filter(tx => ...)`.

Reply:
SGTM